warehouse/datalake/datalake_test.go

```diff
@@ -246,10 +201,5 @@ func TestDatalakeConfigurationValidation(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
-	handle = &TestHandle{
-		S3DatalakeWriteKey:    "ZapZJHfSxUN96GTIuShnz6bv0zi",
-		GCSDatalakeWriteKey:   "9zZFfcRqr2LpwerxICilhQmMybn",
-		AZUREDatalakeWriteKey: "Hf4GTz4OiufmUqR1cq6KIeguOdC",
-	}
-	os.Exit(testhelper.Run(m, handle))
+	os.Exit(testhelper.Run(m, &TestHandle{}))
```

Comment:
We can probably get rid of `TestMain`

Reply:
@lvrach Removed module-level test dependency.